components/coprocessor_plugin_api/build.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+fn main() {
+    println!("cargo:rerun-if-changed=*");
+    println!("cargo:rustc-env=API_VERSION={}", env!("CARGO_PKG_VERSION"));
```

Comment:
If we intended to get "the version of the coprocessor_plugin_api crate the plugin was compiled with", we should move `env!("CARGO_PKG_VERSION")` inside the main program. According to https://doc.rust-lang.org/cargo/reference/environment-variables.html , seems that this env variable is not expected to be read in `build.rs`.

Reply:
Seems true...

Reply:
@Jibbow PTAL~

Reply:
I’ve tested and could get correct version number in build.rs, but this seems not to be documented...